h/api/search.py

```diff
@@ -31,7 +34,41 @@ def _match_clause_for_uri(uristr):
     }
 
 
-def build_query(request_params):
+def scan(es_client, query, fields):
```

Comment:
This doesn't seem to add anything at all to `helpers.scan`. You could just export `helpers.scan` here if you wanted to. Or maybe you're deliberately trying to hide the extra functionality of `helpers.scan`? Any reason?